KomaMRICore/src/ui/DisplayFunctions.jl

```diff
@@ -124,13 +124,13 @@ plot_seq(seq::Sequence; width=nothing, height=nothing, slider=true, show_seq_blo
 	# Visually check the simulation blocks
 	if show_sim_blocks
 		#This is the preparation of the default simulate function
-		t, _ = KomaMRI.get_uniform_times(seq, 1e-3)
-		breaks = KomaMRI.get_breaks_in_RF_key_points(seq,t)
+		t, _ = KomaMRICore.get_uniform_times(seq, 1e-3)
+		breaks = KomaMRICore.get_breaks_in_RF_key_points(seq,t)
 		Nt = length(t)
 		if Nblocks == 0
 			Nblocks = ceil(Int, 6506*Nt/1.15e6)
 		end
```

Comment:
I think these part is deprecated.

Reply:
It is still necessary to avoid division by zero.

Reply:
I meant this `Nblocks = ceil(Int, 6506*Nt/1.15e6)`, which is no longer the behavior when the number of blocks is not specified.